src/internal/target-runner.csx

```diff
@@ -9,51 +10,58 @@ using static SimpleTargetsUtil;
 
 public static class SimpleTargetsTargetRunner
 {
-    public static void Run(IList<string> targetNames, bool dryRun, IDictionary<string, Target> targets, TextWriter output, bool color)
+    public static void Run(IList<string> targetNames, SimpleTargetsTargetRunnerOptions options)

```

Comment:
The targets dictionary and output are not options, they are mandatory. I think the options type should be restricted to dry run, color, and skip dependencies.

Reply:
Will do.